content-src/asrouter/templates/ReturnToAMO/_ReturnToAMO.scss

```diff
@@ -0,0 +1,131 @@
+.ReturnToAMOOverlay,
+.amo + body.hide-main {  // sass-lint:disable-line no-qualifying-elements
+  background: $white;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  display: block;
+  z-index: 2100;
+
+  .ReturnToAMOOuterContainer {
+    width: 835px;
+    height: 400px;
+    position: fixed;
+    left: calc(50% - 417px); // halfway across minus half the width of the modal
+  }
+
+  .ReturnToAMOText {
+    color: $grey-90-80;
+    line-height: 32px;
+    font-size: 23px;
+    width: 100%;
+
+    img {
+      margin-inline-start: 6px;
+      margin-inline-end: 6px;
+    }
+  }
+
+  h2 {
+    color: $grey-60-60;
+    font-weight: 100;
+    margin-bottom: 22px;
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: 1.2px;
+    margin-top: 70px;
+  }
+
+  p {
+    color: $grey-90-50;
+    font-size: 14px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  .puffy {
```

Comment:
ok `.puffy`,`.blue`, `.default` and `.grey` and button styles for these buttons according to the spec and https://design.firefox.com/photon/components/buttons.html. They are every so slightly different than the regular buttons in activity stream or as router so they needed their own styles